README.md

```diff
@@ -113,6 +113,66 @@ next-page-title: "Create a Layout"
 Omit the "prev-page" info for the first page, and the "next-page" info for the
 last page.
 
+## Syntax highlighting
+
+The flutter.io website uses [prism.js](http://prismjs.com/) for syntax
+highlighting. This section covers how to use syntax highlighting, and
+how to update our syntax highlighter for new languages.
+
+### Using syntax highlighting
+
+The easiest way to syntax highlight a block of code is to wrap
+it with `{% prettify _language_ %}`:
+
+```
+{% prettify dart %}
```

Comment:
I'll mention you can do both. But, if you use triple-backtick, you lose some features such as inline highlighting and inline strike.